src/annotator/config.js

```diff
@@ -47,7 +47,7 @@ function configFrom(window_) {
   //
   // In environments where the config has not been injected into the DOM,
   // we try to retrieve it from the URL here.
-  var directLinkedID = annotationQuery.extractAnnotationQuery(window_.location.href);
+  var directLinkedID = extractAnnotationQuery(window_.location.href);
```

Comment:
The function used to return a string indicating the annotation to select when the sidebar opened, it was later altered to return an object of this form:

```ts
{
  // Query string for sidebar's search box
  query: string;
  // Annotation ID to select
  annotations: string;
}
```

The variable name here never got updated.